dist/images/ovndb-raft-functions.sh

```diff
@@ -29,9 +29,9 @@ ready_to_join_cluster() {
   if [[ $? != 0 ]]; then
     return 1
   fi
-  target=$(ovn-${db}ctl --db=${transport}:${init_ip}:${port} ${ovndb_ctl_ssl_opts} --data=bare --no-headings \
+  target=$(ovn-${db}ctl --db=${transport}:[${init_ip}]:${port} ${ovndb_ctl_ssl_opts} --data=bare --no-headings \
     --columns=target list connection)
-  if [[ "${target}" != "p${transport}:${port}" ]]; then
+  if [[ "${target}" != "p${transport}:${port}${ovn_raft_conn_ip}" ]]; then
```

Comment:
this looks awkward because it makes me think this is an IP based on the variable name. Maybe something like ovn_raft_conn_ip_url_suffix or something would be better.